benchexec/tools/validation_utils.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains some useful functions for handling Validation Tasks
```

Comment:
All public functions here need documentation.

Reply:
Sorry must have missed adding it, have added documentation to every function.